Packs/AWS_Bedrock/Integrations/AwsBedrock/AwsBedrock.py

```diff
@@ -0,0 +1,78 @@
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+
+
+def ask_command(bedrock, args):
+
+    question = args.get('question')
+    body = json.dumps({
+        "prompt": f"\n\nHuman:{question}\n\nAssistant:",
+        "max_tokens_to_sample": 30,
+        "temperature": 0.1,
+        "top_p": 0.9,
+    })
+
+    modelId = args.get("model")
+    accept = 'application/json'
+    contentType = 'application/json'
+
+    response = bedrock.invoke_model(body=body, modelId=modelId, accept=accept, contentType=contentType)
+
+    response_body = json.loads(response.get('body').read())
+
+    return CommandResults(
+        outputs_prefix='AWS.Bedrock',
+        readable_output=f'AWS Bedrock API returns with HTTP Code '
+                        f'{response.get("ResponseMetadata").get("HTTPStatusCode")}\n{response_body.get("completion")}',
+        outputs=json.loads(response_body.get("completion")),
+        raw_response=response_body
+    )
+
+
+def test_function(aws_client):
+    client = aws_client.aws_session(service='bedrock')
+
+    response = client.list_foundation_models()
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results('ok')
+
+
+def main() -> None:
+
+    params = demisto.params()
+    command = demisto.command()
+    args = demisto.args()
+
+    # AWS Config
+    aws_default_region = params.get('defaultRegion')
+    aws_role_arn = params.get('roleArn')
+    aws_role_session_name = params.get('roleSessionName')
+    aws_role_session_duration = params.get('sessionDuration')
+    aws_role_policy = None
```

Comment:
Why the aws_role_policy is None?

Reply:
This is declared in the AWSClient as a required argument (palo developed). Other areas such as lambda where Philippe linked to declare it as None.